### ISSUE TITLE
TypeError on Windows with lando 3.9.0

### ISSUE THREAD

**Issue by oskarmodig** [lando/lando#3500]
I just updated to lando 3.9.0 (from 3.6.5) and when I try to start (or rebuild) any app I get this error:
```
TypeError: Cannot read properties of undefined (reading 'confirm')
    at C:\snapshot\cli\node_modules\@lando\core-next\tasks\destroy.js
    at C:\snapshot\cli\node_modules\@lando\core\lib\lando.js
From previous event:
    at processImmediate (node:internal/timers:466:21)
```
As seen in the attached picture it first outputs the help information for the start command, and then fails with the error.

I'm running on Windows 11. Docker Desktop didn't automatically update for me when running the lando installer (I tried twice), so I had to install 4.16.1 manually. I got the same error both before and after updating Docker though.

<img width="823" alt="image" src="https://user-images.githubusercontent.com/3178877/213476571-0a3f2b8b-ff90-4b09-b85b-88a7e828d8db.png">

I also tried running it with the `--clean` command, which seemed to success ("Lando has cleared the tasks cache!"), after which the error changed to not show the help section any more:
<img width="831" alt="image" src="https://user-images.githubusercontent.com/3178877/213477076-22be5777-2a1e-4e3f-a5b3-043224bef55c.png">

**Comment by iKlsR** [comment]
Super fresh install of windows on a new disk and then docker and lando. Getting the same.

**Comment by pirog** [comment]
Thanks for the report. Will look into it!

**Comment by oskarmodig** [comment]
@pirog Thank you! I can add that this is not limited to the `lando start` command, but any lando command I try to run. Including running just `lando`.

**Comment by pirog** [comment]
Yup! I can replicate although not quite sure why the behavior is different on windows yet.
